src/test/java/no/unit/nva/doi/transformer/model/external/DateTypeTest.java

```diff
@@ -0,0 +1,16 @@
+package no.unit.nva.doi.transformer.model.external;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+import no.unit.nva.doi.transformer.model.internal.external.DateType;
+import org.junit.jupiter.api.Test;
+
+public class DateTypeTest {
+
+    @Test
+    public void toStringShouldReturnTheNameOfTheDateType() {
```

Comment:
While it is true that we only use one, should we test all using the enum::values method?